### PR TITLE
Add & clarify acceptance criteria for freeform governance proposals

### LINKED ISSUE #818: 0028-GOVE-governance.md needs specific acceptance criteria for freeform gov?
Currently the spec details governance for both network parameters and freeform.

Can it be clear as to which ACs are common between the sub-features and which would be specific to each.

### PR DESCRIPTION
**Pull request by edd**
Freeform governance proposals were added to the Governance spec (0028-GOVE), but no specific Acceptance Criteria were added, and some of the existing Acceptance Criteria are worded in a way that means they don't apply well to Freeform Governance Proposals. This PR tidies that up

- Add Freeform Governance Proposal network params to 0054-NETP
- Add ACs 0028-GOVE-019 to 0028-GOVE-023
- Reword 0028-GOVE-001
- Reorganise ACs so that it's more obvious which apply to all and which are specific to each type 

Closes #818

### CODE REVIEW

**Comment by edd** [comment]
@MuthuVega This is now ready - let me know if this doesn't clear up your questions

File: protocol/0028-GOVE-governance.md [hunk 1]
```diff
@@ -18,11 +18,11 @@ Governance actions can be the end result of a passed proposal. The allowable typ
...
 1. Create a new market
-1. Change an existing market's parameters
-1. Change network parameters
-1. Add an external asset to Vega (covered in a [separate spec - see 0027](./0027-ASSP-asset_proposal.md))
-1. Authorise a transfer to or from the [Network Treasury](TODO: LINK)
-
+2. Change an existing market's parameters
+3. Change network parameters
+4. Add an external asset to Vega (covered in a [separate spec - see 0027](./0027-ASSP-asset_proposal.md))
+5. Authorise a transfer to or from the [Network Treasury](TODO: LINK)
```
Comment (gordsport) [inline protocol/0028-GOVE-governance.md]: I know not part of this change but the agreed network treasury simplification is going into https://github.com/vegaprotocol/specs-internal/blob/master/protocol/0055-TREA-on_chain_treasury.md 

Wondered if this is correct we could update this link?

Comment (gordsport) [reply protocol/0028-GOVE-governance.md]: maybe it's too soon?

**Review by gordsport — DISMISSED** [review]
Looks good to me clears up what is common, network param and freeform  governance related. Best for @MuthuVega to check too as he is writing the tests

**Review by MuthuVega — DISMISSED** [review]
The Ac for free form governance look good , I shall check these against the tests and fill any gaps. Thx

File: protocol/0028-GOVE-governance.md (hunk 1 above)
Comment (edd) [reply protocol/0028-GOVE-governance.md]: It's on master, so I'll fix the link. It's better than a dead link!
